### PR TITLE
Episode 16

### PR DESCRIPTION
**Pull request by sockeqwe**
Feed validation:
http://castfeedvalidator.com/?url=https://raw.githubusercontent.com/artem-zinnatullin/TheContext-Podcast/episode16/feed.rss

TODO

- [ ] Upload mp3 to server @artem-zinnatullin 
- [ ] Add Git Tag
- [ ] Upload mp3 into release section (once tagged)

### CODE REVIEW

File: feed.rss [hunk 1]
```diff
@@ -1091,5 +1091,63 @@
...
+                    <li><a href="https://github.com/artem-zinnatullin/TheContext-Podcast/issues/86">Discussion after the episode</a></li>
+                    </ul>
+                    
+                    <p>In this episode Artem, Artur and Hannes discuss the tools they use for development.</p>
+                    
+                    <h4>Hosts:</h4>
+                    
+                    <ul>
+                    <li>Artem Zinnatullin <a href="https://twitter.com/artem_zin">@artem_zin</a>, <a href="http://artemzin.com">personal blog</a>, <a href="https://github.com/artem-zinnatullin">GitHub</a>
+                    <ul><li>Hannes Dorfmann <a href="https://twitter.com/sockeqwe">@sockeqwe</a>, <a href="http://hannesdorfmann.com">personal blog</a>, <a href="https://github.com/sockeqwe">GitHub</a></li>
```
Comment (ming13) [inline feed.rss]: Something is off with the `<ul>` tag here.

File: feed.rss [hunk 2]
```diff
@@ -1091,5 +1091,63 @@
...
+                    
+                    <p>Tools:</p>
+                    
+                    <ul>
+                    <li><a href="https://github.com/gojuno/mainframer">Mainframer</a></li>
+                    <li><a href="https://github.com/gojuno/composer">Composer</a></li>
+                    <li><a href="https://github.com/gojuno/swarmer">Swarmer</a></li>
+                    <li><a href="http://ohmyz.sh">zsh</a></li>
+                    <li><a href="https://github.com/AlDanial/cloc">cloc</a></li>
+                    </ul>
```
Comment (ming13) [inline feed.rss]: * https://en.wikipedia.org/wiki/Ncdu
* https://jonas.github.io/tig/
* https://geoff.greer.fm/ag/

File: feed.rss [hunk 3]
```diff
@@ -1091,5 +1091,63 @@
...
+                    
+                    <p>CI:</p>
+                    
+                    <ul>
+                    <li><a href="https://jenkins.io">Jenkins</a></li>
+                    <li><a href="http://circleci.com">Circle.Ci</a></li>
```
Comment (ming13) [inline feed.rss]: I think they call themselves `CircleCI`.

File: feed.rss [hunk 4]
```diff
@@ -1091,5 +1091,63 @@
...
+                    <li><a href="https://drone.io">Drone</a></li>
+                    </ul>
+                    
+                    <p>Libraries:</p>
+                    
+                    <ul>
+                    <li><a href="http://spekframework.org/">Spek</a></li>
+                    <li><a href="https://github.com/JakeWharton/RxReplayingSharel">RxReplayingShare</a></li>
```
Comment (ming13) [inline feed.rss]: `l` in the end of the link.

File: show_notes/Episode_16.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+ - [Discussion after the episode](https://github.com/artem-zinnatullin/TheContext-Podcast/issues/86)
+
+In this episode Artem, Artur and Hannes discuss the tools they use for development.
+
+#### Hosts:
+
+ - Artem Zinnatullin [@artem_zin](https://twitter.com/artem_zin), [personal blog](http://artemzin.com), [GitHub](https://github.com/artem-zinnatullin)
+  - Hannes Dorfmann [@sockeqwe](https://twitter.com/sockeqwe), [personal blog](http://hannesdorfmann.com), [GitHub](https://github.com/sockeqwe)
+  - Artur Dryomov [@arturdryomov](https://twitter.com/arturdryomov), [personal blog](http://arturdryomov.online/), [GitHub](https://github.com/ming13)
```
Comment (ming13) [inline show_notes/Episode_16.md]: Indentation went off a bit.

File: show_notes/Episode_16.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  - [Drone](https://drone.io)
+
+
+Libraries:
+
+  - [Spek](http://spekframework.org/)
+  - [RxReplayingShare](https://github.com/JakeWharton/RxReplayingSharel)
+  - [Moshi](https://github.com/square/moshi)
+
+
```
Comment (ming13) [inline show_notes/Episode_16.md]: Too much!

File: show_notes/Episode_16.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  
+  - [Jenkins](https://jenkins.io)
+  - [Circle.Ci](http://circleci.com)
+  - [Drone](https://drone.io)
+
+
+Libraries:
+
+  - [Spek](http://spekframework.org/)
+  - [RxReplayingShare](https://github.com/JakeWharton/RxReplayingSharel)
```
Comment (ming13) [inline show_notes/Episode_16.md]: XML comments can be applied here as well.

Comment (sockeqwe) [reply show_notes/Episode_16.md]: What do you mean?

**Comment by sockeqwe** [comment]
Thanks @ming13  and @vanniktech

File: feed.rss [hunk 8]
```diff
@@ -1091,5 +1091,66 @@
...
+                    
+                    <p>Tools:</p>
+                    
+                    <ul>
+                    <li><a href="https://github.com/gojuno/mainframer">Mainframer</a></li>
+                    <li><a href="https://github.com/gojuno/composer">Composer</a></li>
+                    <li><a href="https://github.com/gojuno/swarmer">Swarmer</a></li>
+                    <li><a href="http://ohmyz.sh">zsh</a></li>
+                    <li><a href="https://github.com/AlDanial/cloc">cloc</a></li>
+                    <li><a href="https://en.wikipedia.org/wiki/Ncdu">Ncdu</a></li>
```
Comment (ming13) [inline feed.rss]: nit: `ncdu`

**Comment by sockeqwe** [comment]
Updated: ncdu (lower case letter)

**Pull request merged**